.coveragerc

```diff
@@ -334,6 +334,8 @@ omit =
     homeassistant/components/environment_canada/weather.py
     homeassistant/components/envisalink/*
     homeassistant/components/ephember/climate.py
+    homeassistant/components/eps/__init__.py
+    homeassistant/components/eps/alarm_control_panel.py
```

Comment:
I'd work on adding these tests if you have time. More likely to get your PR approved quicker if you have better coverage